### DISCUSSION TITLE
Loadouts and New Components

### DISCUSSION THREAD

**Discussion by bostonmyk** [PygmalionOfCyprus/cmo-db-requests#28] [Q&A] [answered]
Hey Guys

How do you want to handle loadouts with new pods, weapons etc. ? Two entries. One for the Component and one for a loadout or are you ok with 1 loadout request with all the stuff in it?

Mike

**Comment by PygmalionOfCyprus** [comment, marked as answer]
For the moment, you can just submit a loadout request with the new components listed. Most of the time we can build new weapons, sensors pods, etc. quite quickly even if they aren't already in the DB (especially if they're variants/based on extant weapons), so you don't need to go through the time of filing a separate component request.

If we encounter a component that needs deeper research for us to add it as part of the loadout, we'll let you know and at that time you can file a separate weapon/sensor request.
